dex/config/config_test.go

```diff
@@ -262,3 +262,60 @@ func DataConversion(t *testing.T) {
 		t.Fatalf("map-cfg-obh: expected value for key 'key5' not to change, changed to '%s'", cfg.Key5)
 	}
 }
+
+func TestParseIntoEmbeddedStructs(t *testing.T) {
+	type BB struct {
+		B int `ini:"b"`
+	}
+
+	settings := map[string]string{
+		"a": "1",
+		"b": "2",
+	}
+
+	// Embedded value.
+	cfg := &struct {
+		A int `ini:"a"`
+		BB
```

Comment:
I think the `ini` package supports this out of the box with `ini:",extends"` as the tag.

i.e.

```go
cfg := &struct {
	A  int `ini:"a"`
	BB `ini:",extends"`
}{}
```

and we can keep `ParseInto` unchanged, calling `section.MapTo(obj)` once directly.

Reply:
How did you find this?

Reply:
Had a vague recollection of https://github.com/go-ini/ini/commit/5e97220809ffaa826f787728501264e9114cb834 while reviewing the commits when I bummed it to 1.62 in https://github.com/decred/dcrdex/pull/1606/commits/9c461e34c72b3446950f495f6196e2a1a31544f7#diff-33ef32bf6c23acb95f5902d7097b7a1d5128ca061167ec0716715b0b9eeaa5f6R51

Reply:
I do kinda with they would have just done it my way.

Reply:
I hear you about the that.  Not sure why a tag is needed.

Regardless, I don't think this is the kind of thing we should get side tracked with even if they didn't have this tag. It's more than fine to just call `Unmapify` twice, once on the parent and once on the embedded field.  Either that or we just don't compose a data-only struct like this with embedding to avoid extra complexity of the pre-processing.  A little repetition to avoid complexity and non-trivial code to maintain is fine.  The flatten function with the reflection code calls `section.MapTo` repeatedly anyway right? 

New issue because I think we have some unhealthy patterns established that create the need for these acrobatics in the first place: https://github.com/decred/dcrdex/issues/1698

Reply:
I've used the ``ini:",extends"`` tag. Do you not want to use it?

Reply:
No please use it!  I'm just thinking we need to look to fix our use of Unmapify and the ini package in the future.